pocs/core.py

```diff
@@ -298,12 +298,20 @@ def is_safe(self, no_warning=False):
 
         is_safe_values = dict()
 
+        # Check if AC power connected and return immediately if not
```

Comment:
The idea was that if power is out you don't want to waste time/power checking the weather or anything so that it is conserving battery.

Reply:
Well, that is true, but the same might be said of weather too. Unless we have evidence otherwise, I think these will all happen fairly fast so we don't need to worry about the short circuit.

OTOH, I'd like to head to a situation in which there is no db look up or call at all. We accumulate reports from various sensors and can periodically turn that into a health report, including dealing with stale reports. But, that is for later.